cmake/reports/CMakeLists.txt

```diff
@@ -17,12 +17,4 @@ foreach(report ram_report rom_report)
     )
 endforeach()
 
-add_custom_command(
-  TARGET rom_report
-  POST_BUILD
-  COMMAND
-  ${PYTHON_EXECUTABLE}
-  ${ZEPHYR_BASE}/../nrf/scripts/partition_manager_report.py
-  --input ${CMAKE_BINARY_DIR}/partitions.yml
-  "$<$<NOT:$<TARGET_EXISTS:partition_manager>>:--quiet>"
-  )
+include(${ZEPHYR_BASE}/../nrf/cmake/reports.cmake)
```

Comment:
I think we have a variable for ${ZEPHYR_BASE}/../nrf now

Reply:
I'm not able to find it. I copied what is used in boilerplate.cmake

Reply:
ZEPHYR_NRF_MODULE_DIR: /home/sebo/ncs/nrf

Reply:
THis will be made available after upmerge, will wait for this anyway.